controllers/forwarding.js

```diff
@@ -1,9 +1,23 @@
-const {asyncHandler} = require('./utils.js');
+const axios = require('axios');
 
-// takes a remote url for foreign service for deployment and a local port in development
+// takes a string for the env var name for deployment and a portNumber in development
 // returns endpoint that forwards request to either remote or local args
-exports.forwardingController = (remote = 'forward requres here', local ='fallback port if local') => (req, res, next) => {
-  //TODO handle forwarding, body, headers, routes
-  console.log('forwarding controller')
-  res.json(`${req.method}@${req.originalUrl} works. remote: ${ remote } local: ${ local }`); // test to make sure it works
-}
\ No newline at end of file
+exports.forwardingController = (envVarName = 'ENV var for service', portNumber ='port') => async (req, res, next) => {
```

Comment:
Never seen a double assigning to a function like this:

_const variable_name = (params) => (more_params) => {
  // code
}_

Pretty cool, I'll have to look it up/check it out

Reply:
comes in handy. the higher order functions and currying are a game changer